DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -232,10 +238,155 @@ public IActionResult ConfirmDeleteGroup(int groupId, ConfirmDeleteGroupViewModel
             return RedirectToAction("Index");
         }
 
+        [HttpGet]
+        [Route("Generate")]

```

Comment:
These can be combined into [HttpGet("route string")], same for the Post

Reply:
done